content/_code-samples/key-derivation/js/README.md

```diff
@@ -0,0 +1,24 @@
+# JavaScript key derivation examples
+
+Generates key from a given input in Ed25519 and Secp256k1 format. On first run, you
+have to install the necessary node.js dedpendencies:
+
+    npm install
+
+## Command-line usage:
+
+### Password like seed    
+
+    npm start "sEdSKaCy2JT7JaM7v95H9SxkhP9wS2r"
+
+### Rippled RFC1751 Mnemonic
+
+    npm start "KANT FISH GENE COST WEB JAKE CHUM HAD SOD MID KICK BOTH"
+
+### Hex formatted seed
+
+    npm start "0x21edc3dec3ef1873cf8f333381c5f360c3"
```

Comment:
Currently when I run this the snippet crashes: 

```
    if (!val) throw new Error(msg || 'Assertion failed');
              ^

Error: Invalid character in 0x21edc3dec3ef1873cf8f333381c5f360c3
    at assert (/Users/jacksonmills/Documents/node_modules/bn.js/lib/bn.js:6:21)
    at parseHex4Bits (/Users/jacksonmills/Documents/node_modules/bn.js/lib/bn.js:198:7)
    at parseHexByte (/Users/jacksonmills/Documents/node_modules/bn.js/lib/bn.js:203:13)
    at BN._parseHex (/Users/jacksonmills/Documents/node_modules/bn.js/lib/bn.js:225:13)
    at BN.init [as _init] (/Users/jacksonmills/Documents/node_modules/bn.js/lib/bn.js:103:14)
    at new BN (/Users/jacksonmills/Documents/node_modules/bn.js/lib/bn.js:39:12)
    at hexToBytes (/Users/jacksonmills/Documents/xrpl-dev-portal-bounty-alex/xrpl-dev-portal/test.js:31:36)
    at Seed._initSeed (/Users/jacksonmills/Documents/xrpl-dev-portal-bounty-alex/xrpl-dev-portal/test.js:60:29)
    at new Seed (/Users/jacksonmills/Documents/xrpl-dev-portal-bounty-alex/xrpl-dev-portal/test.js:45:27)
    at Object.<anonymous> (/Users/jacksonmills/Documents/xrpl-dev-portal-bounty-alex/xrpl-dev-portal/test.js:195:18)
```

Reply:
Fixed, I removed the "0x" and now it works flawlessly. Probably a copy-paste error from PHP code, been switching back and forth from PHP to JavaScript.